lms/views/admin/application_instance.py

```diff
@@ -265,6 +265,10 @@ def show_instance(self):
     def update_instance(self):
         ai = self._get_ai_or_404(**self.request.matchdict)
 
+        for ai_field in ["lms_url", "custom_canvas_api_domain", "deployment_id"]:
+            if value := self.request.params.get(ai_field):
+                setattr(ai, ai_field, value.strip())
```

Comment:
```suggestion
            if value := self.request.params.get(ai_field, "").strip():
                setattr(ai, ai_field, value)
```

Might be a better way than this, but a string containing only spaces will be truthy until stripped.

Reply:
Is there any uniqueness check or anything else for `deployment_id`?